mount.go

```diff
@@ -23,6 +24,8 @@ const (
 	devPrefix      = "/dev/"
 	timeoutHotplug = 3
 	mountPerm      = os.FileMode(0755)
+
+	driver9pType = "9p"
```

Comment:
nits: please define it together with other types like `driverBlkType` and `driverSCSIType`.

Reply:
You would prefer that I put them all inside device.go ?

Reply:
Or any other place you feel comfortable with. I just want them to be put together.

Reply:
Ok fair enough. Thanks for the review, I'll fix unit tests on Monday.